components/viaduct/android/src/main/java/mozilla/appservices/httpconfig/HttpConfig.kt

```diff
@@ -95,8 +96,8 @@ object RustHttpConfig {
                     }
                     rb
                 } catch (e: Throwable) {
-                    LibViaduct.INSTANCE.viaduct_log_error("Network error: ${e.message}")
-                    MsgTypes.Response.newBuilder().setExceptionMessage(e.message)
+                    LibViaduct.INSTANCE.viaduct_log_error("Network error: ${Log.getStackTraceString(e)}")
```

Comment:
Not opposed to this but does it have any PII concerns?

Reply:
It should be fine: it logs a classic Java stack trace (no parameters or any kind of PII as far as I saw).

Reply:
Stacktraces could easily have PII embedded in an exception message, and the new version has a higher chance of hitting that, I think? Depending on how exactly stacktrace is printed out.

Reply:
Reverted the file.